src/cljs/vanilla/widgets/carousel.cljs

```diff
@@ -295,10 +295,10 @@
   ; TODO: this is a hack for the following hack (does NOT unsubscribe to sources when widget closes)
   (ds/data-source-subscribe [:spectrum-traces :power-data])
```

Comment:
need to update the subscriptions to be [:channel-power-1000-service :channel-power-2000-service :channel-power-3000-service :channel-power-4000-service]